utils/parse_test.go

```diff
@@ -8,34 +8,47 @@ import (
 
 func TestParseShell(t *testing.T) {
 	cases := map[string][]string{
-		"":                  []string{},
-		"one":               []string{"one"},
-		"one two":           []string{"one", "two"},
-		"\"one\"":           []string{"one"},
-		"'one'":             []string{"one"},
-		"one two three":     []string{"one", "two", "three"},
-		"one \"two\" three": []string{"one", "two", "three"},
-		"one \"two three\"": []string{"one", "two three"},
-		"one 'two three'":   []string{"one", "two three"},
-		"\"one two three\"": []string{"one two three"},
-		"'one two three'":   []string{"one two three"},
-		"‘one two three’":   []string{"one two three"},
-		"“one two three”":   []string{"one two three"},
-		"one 'two three":    []string{"one 'two three"},
+		"":                           []string{},
+		"one":                        []string{"one"},
+		"one two":                    []string{"one", "two"},
+		"\"one\"":                    []string{"one"},
+		"'one'":                      []string{"'one'"},
+		"one two three":              []string{"one", "two", "three"},
+		"one \"two\" three":          []string{"one", "two", "three"},
+		"one \"two three\"":          []string{"one", "two three"},
+		"one 'two three'":            []string{"one", "'two", "three'"},
+		"\"one two three\"":          []string{"one two three"},
+		"'one two three'":            []string{"'one", "two", "three'"},
+		"‘one two three’":            []string{"'one", "two", "three'"},
```

Comment:
[nit] this test isn't adding much since the case above it is functionally identical